agent_verification/Cargo.toml

```diff
@@ -20,19 +20,22 @@ oid4vp.workspace = true
 reqwest.workspace = true
 serde.workspace = true
 serde_json.workspace = true
+siopv2.workspace = true
 strum.workspace = true
 strum_macros.workspace = true
-siopv2.workspace = true
 thiserror.workspace = true
 tracing.workspace = true
-url.workspace = true
 tokio.workspace = true
+url.workspace = true
 
 [dev-dependencies]
+agent_secret_manager = { path = "../agent_secret_manager", features = [
+    "test_utils",
+] }
 agent_shared = { path = "../agent_shared", features = ["test_utils"] }
 agent_verification = { path = ".", features = ["test_utils"] }
-
```

Comment:
This line is usually left in to separate "internal" from "external" dependencies (see `[dependencies]` in this file).